_bibliography/ifb.bib

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+---
+
+% ordered by year, then alphabetically
+
+# Publications
+
+# 2018
```

Comment:
w4m publication?

Reply:
I thought that better none than few not well chosen.

If I add W4M, why not add proteore ... and oh, and you have forgot this one ... blablabla

We have a usegalaxy.fr meeting this week. I can add this point in the agenda.

Reply:
fine with me ... do you think this is ready?

Reply:
Ready 🚀